packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -18,20 +20,15 @@ import i18n from './i18n/i18n';
 export class GcdsCard {
   @Element() el: HTMLElement;
 
-  /**
-   * The type attribute specifies how the card renders as a link
-   */
-  @Prop({ reflect: true }) type: 'link' | 'action' = 'link';
-
   /**
    * The card title attribute specifies the title that appears on the card
    */
   @Prop({ reflect: true }) cardTitle!: string;
```

Comment:
Following the new validation idea: should we stop the component from being rendered if there is no card title since the card title is required?